.github/settings.yml

```diff
@@ -0,0 +1,87 @@
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: scorecard
+
+  # A short description of the repository that will show up on GitHub
+  description: Security Scorecards - Security health metrics for Open Source
+
+  # A URL with more information about the repository
+  homepage: ''
+
+  # Updates the default branch for this repository.
+  default_branch: main
+
+  # Either `true` to enable automated security fixes, or `false` to disable
+  # automated security fixes.
+  enable_automated_security_fixes: true
+
+  # Either `true` to enable vulnerability alerts, or `false` to disable
+  # vulnerability alerts.
+  enable_vulnerability_alerts: true
+
+# See https://docs.github.com/en/rest/reference/teams#add-or-update-team-repository-permissions for available options
+teams: []
+    # The permission to grant the team. Can be one of:
+    # * `pull` - can pull, but not push to or administer this repository.
+    # * `push` - can pull and push, but not administer this repository.
+    # * `admin` - can pull, push and administer this repository.
+    # * `maintain` - Recommended for project managers who need to manage the repository without access to sensitive or destructive actions.
+
+# Collaborators: give specific users access to this repository.
+# See https://docs.github.com/en/rest/reference/collaborators for available options
+
+# THIS LIST SHOULD ALWAYS MATCH TOP-LEVEL CODEOWNERS
+# https://github.com/ossf/scorecard/blob/main/.github/CODEOWNERS
+
+# TODO(owners): For ease of management, this should eventually shift to a
+#               defined GitHub team instead of individual usernames
+collaborators:
+  - name: azeemshaikh38
+    permission: admin
+
+  - name: inferno-chromium
+    permission: admin
+
+  - name: justaugustus
+    permission: admin
+
+  - name: laurentsimon
+    permission: admin
+
+  - name: naveensrinivasan
+    permission: admin
+
+branches:
+  - name: main
+    # https://docs.github.com/en/rest/reference/repos#update-branch-protection
+    # Branch Protection settings. Set to null to disable
+    protection:
+      # Required. Require at least one approving review on a pull request, before merging. Set to null to disable.
+      required_pull_request_reviews:
+        # The number of approvals required. (1-6)
+        required_approving_review_count: 1
+        # Dismiss approved reviews automatically when a new commit is pushed.
+        dismiss_stale_reviews: true
```

Comment:
we have this false right now as it makes it very annoying to do back and forth.

Reply:
+1. We tried `dismiss_stale_reviews` once and it was difficult to work with. Would prefer to keep this as `false`.

Reply:
I agree this should be false.

Reply:
That's fair, but I think we've had such a bad experience with this, I think it would be preferable to keep it off for now. 

I'm open to trying a different version of this (like the one you mentioned being used in Kubernetes). But I would prefer to keep that out of scope for this PR atleast and take it up as a separate issue/discussion.

Reply:
I also like the k8's way of lgtm :-) feature request for GitHub!